manager/base/src/main/java/org/wildfly/security/manager/_private/SecurityMessages.java

```diff
@@ -45,6 +50,10 @@
     @Message(value = "Permission check failed (permission \"%s\" in code source \"%s\" of \"%s\")")
     void accessCheckFailed(Permission permission, CodeSource codeSource, ClassLoader classLoader);
 
+    @LogMessage(level = TRACE)
```

Comment:
It is correct you have updated SecurityMessages to extend BasicLogger but we don't add log message for TRACE level logging, now you have extended BasicLogger you can just call tracef directly.

Reply:
I have updated the pull request: The trace has been removed from SecurityMessages and been placed into WFSM directly.